python/lsst/sims/GalSimInterface/galSimDetector.py

```diff
@@ -58,7 +58,9 @@ def __init__(self, afwDetector, afwCamera, obs_metadata, epoch, photParams=None,
         self.fitsHeader.set("EXTTYPE", "IMAGE")
 
         if self.obs_metadata.bandpass is not None:
-            if not isinstance(self.obs_metadata.bandpass, list) and not isinstance(self.obs_metadata.bandpass, numpy.ndarray):
+            if not isinstance(self.obs_metadata.bandpass, list) and not \
+                isinstance(self.obs_metadata.bandpass, np.ndarray):
+
```

Comment:
Could rewrite above with parentheses starting after "if" in order to avoid backslash.